jsk_topic_tools/src/connection_based_nodelet.cpp

```diff
@@ -52,6 +60,13 @@ namespace jsk_topic_tools
     }
   }
 
+  void ConnectionBasedNodelet::wallTimerCallback(const ros::WallTimerEvent& event)
+  {
+    if (!ever_subscribed_) {
+      ROS_INFO("The node subscribes topics only with child subscribers. (`always_subscribe: true` to force it to subscribe)");
```

Comment:
* Add name of nodelet to the message http://docs.ros.org/jade/api/nodelet/html/classnodelet_1_1Nodelet.html#a07d4a1043b86826ca424df4ff7ce7b0e